### PR TITLE
Update Teaser Text for the Remote Interview Blog Post

### PR DESCRIPTION
**Pull request by georgiee**
small update to the teser & seo text of the new article.

[Preview](https://satellytescommain21751-choreinterviewupdate.gtsb.io/blog/we-work-remotely/)

Looks good in the overview:
Screenshot 2022-03-25 at 07 30 59

Looks good SEO-wise:
Screenshot 2022-03-25 at 07 31 40

### CODE REVIEW

File: blog-posts/return-to-office-interview.md [hunk 1]
```diff
@@ -4,8 +4,9 @@ date: "2022-03-24"
 title: We work remotely!?
 featuredImage: images/return-to-office-interview-header.jpg
 author: "Satellytes"
-authorSummary: "We work remotely!? – Interview mit den Satellytes-Geschäftsführern zur Return to Office Policy"
-teaserText: "Mit dem Ausklingen der Corona-Maßnahmen endet in vielen Unternehmen die Home-Office-Pflicht – auch bei Satellytes. Da drängt sich uns die Frage auf: Wie und wo arbeiten wir in Zukunft? Wir haben die Geschäftsführer gefragt."
+authorSummary: "Interview"
```
Comment (georgiee) [inline blog-posts/return-to-office-interview.md:7]: It did super weird so I shorten it. Removing is not possible.

File: blog-posts/return-to-office-interview.md [hunk 2]
```diff
@@ -4,8 +4,9 @@ date: "2022-03-24"
...
+teaserText: Vollzeit im Office? Nur noch Remote? Oder ein bisschen von beidem? Wir haben die Geschäftsführer von Satellytes gefragt.
+seoMetaText: Vollzeit im Office? Nur noch Remote? Oder ein bisschen von beidem? Wir haben die Geschäftsführer von Satellytes gefragt.
```
Comment (georgiee) [inline blog-posts/return-to-office-interview.md:9]: SEO was missing and the excerpt was picked instead which is wrong.

File: blog-posts/return-to-office-interview.md [hunk 3]
```diff
@@ -4,8 +4,9 @@ date: "2022-03-24"
 title: We work remotely!?
 featuredImage: images/return-to-office-interview-header.jpg
 author: "Satellytes"
-authorSummary: "We work remotely!? – Interview mit den Satellytes-Geschäftsführern zur Return to Office Policy"
-teaserText: "Mit dem Ausklingen der Corona-Maßnahmen endet in vielen Unternehmen die Home-Office-Pflicht – auch bei Satellytes. Da drängt sich uns die Frage auf: Wie und wo arbeiten wir in Zukunft? Wir haben die Geschäftsführer gefragt."
+authorSummary: "Interview"
+teaserText: Vollzeit im Office? Nur noch Remote? Oder ein bisschen von beidem? Wir haben die Geschäftsführer von Satellytes gefragt.
```
Comment (georgiee) [inline blog-posts/return-to-office-interview.md:8]: new text, less relation to the law which is not our motivation

**Comment by georgiee** [comment]
again gatsby deployment is stuck, I wonder why this is happening? or just a queue?

**Review by chunksnbits — APPROVED** [review]

File: blog-posts/return-to-office-interview.md (hunk 3 above)
Comment (chunksnbits) [reply blog-posts/return-to-office-interview.md:8]: Sounds good. Nice job! 🎉

**Comment by georgiee** [comment]
Thanks for the approval @chunksnbits

**Pull request merged**